### PR TITLE
Fix for DOC-372 with clarification of COMMAND_CLASS information…

### PR DESCRIPTION
**Pull request by fiowro**
...and minor formatting fixes

### CODE REVIEW

File: doc/source/management/audit_log_plugin.rst [hunk 1]
```diff
@@ -40,8 +40,12 @@ Example of the Disconnect event: ::
...
+* **Query** - Additional fields for this event are: ``COMMAND_CLASS`` (values come from the ``com_status_vars`` array in the :file:`sql/mysqld.cc`` file in a MySQL source distribution. Examples are ``select``, ``alter_table``, ``create_table``, ``show_errors``, etc.), ``CONNECTION_ID``, ``STATUS`` (indicates error when non-zero), ``SQLTEXT`` (text of SQL-statement), ``USER``, ``HOST``, ``OS_USER``, ``IP``. Possible values for the ``NAME`` name field for this event are ``Query``, ``Prepare``, ``Execute``, ``Change user``, etc.
 
+.. note::
+ 
+   It may be not obvious, but ``COMMAND_CLASS="error"`` entry in the audit logs is triggered by ``show_error`` value, and just means that errors are allowed to be shown.
```
Comment (laurynas-biveinis) [inline doc/source/management/audit_log_plugin.rst]: What is the source for this statement?

Comment (fiowro) [reply doc/source/management/audit_log_plugin.rst]: Hm... Just maillist thread linked to the issue. Yes, I see I could be overoptimistic, taking a guess for the resolution. Do you think it's better to remove the note?

Comment (laurynas-biveinis) [reply doc/source/management/audit_log_plugin.rst]: With the note removed, is there any content remaining in the PR?

Comment (fiowro) [reply doc/source/management/audit_log_plugin.rst]: This PR also contains a pair of formatting/typo fixes

**Review by laurynas-biveinis — COMMENTED** [review]
I only see whitespace changes on line 44 and nothing else?

**Comment by fiowro** [comment]
````
``alter_table", "create_table"   ------>  ``alter_table``, ``create_table``
````

**Review by laurynas-biveinis — APPROVED** [review]
Now I see, thank you

**Comment by laurynas-biveinis** [comment]
Why an extra commit appeared seemingly merging 5.7 to 5.5?

**Comment by fiowro** [comment]
sorry, a typo in command

**Comment by peterschwaller** [comment]
touch

**Pull request merged**